snek/snek.py

```diff
@@ -251,20 +251,23 @@ def __load_sitemap(self):
                 'date': None
             }
 
-            # Load from front-matter and add to default
-            try:
-                metadata_from_file = frontmatter.load(filepath, handler=frontmatter_json_handler()).metadata
-                for key, value in metadata_from_file.items():
-                    metadata[key] = value
-
             # If the file could not be read or open, go to next file
-            except FileNotFoundError:
+            if os.access(filepath, os.R_OK):
```

Comment:
Good catch 👍 !

Reply:
Actually I made a mistake (just committed). Should be `if not`...

Reply:
This would likely need to be `if not os.access(filepath, os.R_OK)` though.